modules/iap/charts/iap/values.yaml

```diff
@@ -6,20 +6,20 @@
 iap: 
   backendConfig:
     name: "iap-config-default"
-    iapSecretName: "my-secret"
 
   secret:
+    name: "iap-secret"
     client_id: ""
     client_secret: ""
 
   managedCertificate:
     name: "iap-managed-cert"
     domain: ""
 
-  jupyterIngress:
-    name: "jupyter-ingress"
+  ingress:
+    name: "iap-ingress"
     staticIpName: "xyz"
-    managedCertificates: "iap-managed-cert"
```

Comment:
why did we delete the cert?

Reply:
It is duplicated with  managedCertificate. name: "iap-managed-cert". They should auto linked together.